cluster/vagrant-openshift/provider.sh

```diff
@@ -20,9 +20,8 @@ function up() {
     scp $OPTIONS master:/usr/local/bin/oc ${KUBEVIRT_PATH}cluster/vagrant-openshift/.oc
     chmod u+x cluster/vagrant-openshift/.oc
 
-    vagrant ssh master -c "sudo cat /etc/origin/master/openshift-master.kubeconfig" >${KUBEVIRT_PATH}cluster/vagrant-openshift/.kubeconfig
-
     # Login to OpenShift
+    export KUBECONFIG=${KUBEVIRT_PATH}cluster/os-3.9.0-alpha.4/.kubeconfig
```

Comment:
do we really want to use the  os-3.9.0-alpha providers .kubeconfig for the vagrant-openshift provider? This doesn't seem right.

Reply:
fixed